services/apps/merge_suggestions_worker/src/activities/member-merge-suggestions/getMergeSuggestions.ts

```diff
@@ -0,0 +1,250 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  IMemberPartialAggregatesOpensearch,
+  IMemberPartialAggregatesOpensearchRawResult,
+  IMemberQueryBody,
+  ISimilarMemberOpensearch,
+} from 'types'
+import { svc } from '../../main'
+import { IMemberMergeSuggestion, OpenSearchIndex } from '@crowd/types'
+import { calculateSimilarity, prefixLength } from 'utils'
+import MemberMergeSuggestionsRepository from 'repo/memberMergeSuggestions.repo'
+
+export async function getMergeSuggestions(
```

Comment:
Just a nit: would be cool to add a comment with exactly what we are checking, so it's easier to understand.